File: h2o-docs/src/booklets/v2_2015/source/DeepLearning_Vignette_code_examples/deeplearning_checkpoint.R
# Re-start the training process on a saved DL model
# using the `checkpoint` argument
model_chkp <- h2o.deeplearning(
        x = x,
        y = y,
        training_frame = train,
        validation_frame = test,
        distribution = "multinomial",
        checkpoint = model@model_id,
        activation = "RectifierWithDropout",
        hidden = c(32,32,32),
        input_dropout_ratio = 0.2,
        sparse = TRUE,
        l1 = 1e-5,
        epochs = 20)

File: h2o-docs/src/booklets/v2_2015/source/DeepLearning_Vignette_code_examples/deeplearning_checkpoint.py
# Re-start the training process on a saved DL model
# using the `checkpoint` argument
model_chkp = h2o.deeplearning(
        x=x,
        y=y,
        training_frame=train,
        validation_frame=test,
        checkpoint=model,
        distribution="multinomial",
        activation="RectifierWithDropout",
        hidden=[200,200,200],
        input_dropout_ratio=0.2,
        sparse=True,
        l1=1e-5,
        epochs=20)
